<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="atchutarao" failures="1" tests="6" name="testA" time="0.105" errors="0" timestamp="2024-12-09T14:40:57 IST">
  <testcase classname="TEST.testmylistener" name="tc1" time="0.018">
    <failure type="java.lang.AssertionError" message="expected [20] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [20] but found [10]
at TEST.testmylistener.tc1(testmylistener.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase classname="TEST.testmylistener" name="tc3" time="0.003"/>
  <testcase classname="TEST.testmylistener" name="tc4" time="0.002"/>
  <testcase classname="TEST.testmylistener" name="tc5" time="0.004"/>
  <testcase classname="TEST.testmylistener" name="tc6" time="0.003"/>
  <testcase classname="TEST.testmylistener" name="tc2" time="0.0">
    <skipped/>
  </testcase> <!-- tc2 -->
</testsuite> <!-- testA -->
